RosettaImmutables/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.rosetta</groupId>
+    <artifactId>Rosetta</artifactId>
+    <version>3.11.11-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>RosettaImmutables</artifactId>
+
+  <properties>
+    <hubspot.module.type>CORE</hubspot.module.type>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.hubspot.immutables</groupId>
+        <artifactId>hubspot-style</artifactId>
+        <version>1.3-SNAPSHOT</version>
```

Comment:
We can't depend on SNAPSHOTs in oss stuff

Reply:
Yep, I need to do a release of hubspot-immutables.  We've been developing against 1.3-SNAPSHOT for 4 years so WireSafeEnum doesn't exist in any released version afaik.